api/controllers/auth.js

```diff
@@ -0,0 +1,46 @@
+const bcrypt = require("bcryptjs");
+const config = require("config");
+
+const { authService } = require("../services");
+
+const {
+  validateLoginData,
+  validateRegisterData,
+} = require("../validation/auth");
+
+const login = async (req, res) => {
+  try {
+    const { email, password } = req.body;
+
+    const errors = validateLoginData(email, password);
+
+    if (Object.keys(errors).length > 0) {
+      return res.json(errors);
+    }
+
+    const response = await authService.login(req.body);
+    res.status(201).send(response);
+  } catch (error) {
+    console.error(error);
+    res.json(error);
```

Comment:
Get rid of try/catch and send the status with errors. Check this notices in register too.